src/java/map/ConcurrentBinaryMask.java

```diff
@@ -301,6 +315,11 @@ public void show() {
         this.binaryMask.show();
     }
 
+    public ConcurrentBinaryMask startVisualDebugger() {
+        this.binaryMask.startVisualDebugger(Util.getStackTraceParentClass());
+        return this;
+    }
+
```

Comment:
same

Reply:
it uses the toString() name from the BinaryMask function I made...